int32 vehicle_id
string class_name

float64 x
float64 y
float64 yaw
float64 v

int32 start_node_id
int32 end_node_id

bool toggle_sim
---
bool success
string message